official/resnet/keras/keras_common.py

```diff
@@ -34,6 +34,14 @@
 TRAIN_TOP_1 = 'training_accuracy_top_1'
 
 
+class BatchTimestamp(object):
+  """A structure to store batch time stamp."""
+
+  def __init__(self, batch_index):
+    self.batch_index = batch_index
+    self.timestamp = time.time()
+
+
 class TimeHistory(tf.keras.callbacks.Callback):
   """Callback for Keras models."""
 
```

Comment:
Can you update the init to accept log_steps that defaults to 100?  We use this in tf_cnn_benchmarks and it is very useful.  We often log every 10.